pkgs/development/tools/build-managers/meson/0.60/gir-fallback-path.patch

```diff
@@ -0,0 +1,18 @@
+diff -Naur meson-0.60.2-old/mesonbuild/modules/gnome.py meson-0.60.2-new/mesonbuild/modules/gnome.py
+--- meson-0.60.2-old/mesonbuild/modules/gnome.py	2021-11-02 17:00:40.000000000 -0300
++++ meson-0.60.2-new/mesonbuild/modules/gnome.py	2021-12-12 19:10:58.205198176 -0300
+@@ -923,6 +923,14 @@
+         if fatal_warnings:
+             scan_command.append('--warn-error')
+ 
++        if len(set([girtarget.get_custom_install_dir()[0] for girtarget in girtargets])) > 1:
```

Comment:
This can try to access an empty list, see https://github.com/NixOS/nixpkgs/pull/144779#discussion_r748594428

Reply:
I changed the patch.